packages/core/src/test/dynamoDb/commands/deleteDynamoDbTable.test.ts

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as sinon from 'sinon'
+import * as assert from 'assert'
+import { DynamoDbClient } from '../../../shared/clients/dynamoDbClient'
+import { deleteDynamoDbTable } from '../../../dynamoDb/commands/deleteDynamoDbTable'
+import { DynamoDbTableNode } from '../../../dynamoDb/explorer/dynamoDbTableNode'
+import { AWSError } from 'aws-sdk'
+import { DeleteTableOutput } from 'aws-sdk/clients/dynamodb'
+import { PromiseResult } from 'aws-sdk/lib/request'
+import * as utilities from '../../../shared/utilities/messages'
+import { DynamoDbInstanceNode } from '../../../dynamoDb/explorer/dynamoDbInstanceNode'
+
+describe('deleteDynamoDbTable', () => {
+    let dynamoDbClient: DynamoDbClient
+    let dynamoDbTableNode: DynamoDbTableNode
+
+    beforeEach(() => {
+        dynamoDbTableNode = sinon.stub() as unknown as DynamoDbTableNode
+        dynamoDbClient = new DynamoDbClient('us-west-2')
+    })
+
+    afterEach(() => {
+        sinon.restore()
+    })
+
+    it('should successfully delete a DynamoDB table', async () => {
```

Comment:
You can drop the successfully here
```suggestion
    it('should delete a DynamoDB table', async () => {
```